services/slack/index.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import type { Event } from '../../server/events';
+
+export default {
```

Comment:
Not sure what we need here but maybe these should be classes that would inherit from a base that would house unified logging/etc. Not sure what else we need to expose but logging feels like an obvious one that we can abstract from 3rd party ones later